pkg/morph/client/notary.go

```diff
@@ -380,11 +380,13 @@ func (c *Client) NotarySignAndInvokeTX(mainTx *transaction.Transaction) error {
 		return err
 	}
 
+	magicNumber, _ := c.MagicNumber()
```

Comment:
Could you add a comment, about why the error is ignored?

Reply:
sure, added